src/app/Caches/CacheMetrics.tsx

```diff
@@ -186,16 +187,16 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
           <TextContent>
             <TextList component={TextListVariants.dl}>
               <TextListItem component={TextListItemVariants.dt}>
-                {displayUtils.formatNumber(stats.current_number_of_entries)}
+                {displayUtils.formatNumber(size)}
               </TextListItem>
               <TextListItem component={TextListItemVariants.dd}>
-                Current number
+                Size
```

Comment:
I think we should be more specific here. Size would seem like you mean the total size in some measurement GB/MB. "Current number of entries" is better than "Size" if that's actually what this label shows.